CrewChiefV4/UserInterface/MainWindow.cs

```diff
@@ -643,6 +643,13 @@ public MainWindow()
             InitializeComponent();
             SetupNotificationTrayIcon();
 
+            if (System.Diagnostics.Debugger.IsAttached)
+            {
+                // Restore last saved trace file name.
+                filenameTextbox.Text = UserSettings.GetUserSettings().getString("last_trace_file_name");
```

Comment:
to clarify, this simply saves effort pasting trace file name every time CC is started.  In a workflow when you debug, fix, debug again, fix etc using same trace :)